value_constraints/dont_know.jsonld

```diff
@@ -0,0 +1,7 @@
+{
+    "@context": [ "https://raw.githubusercontent.com/zaliqarosli/schema-standardization/2019-04-08-AddSubActivities/contexts/generic.jsonld"
+    ],
+    "type": "xsd:string",
+    "schema:name": "Don't Know",
+    "schema:value": "dont_know"
+}
```

Comment:
as the constant `schema.repronim.org/dont_know_answer` hasn't been defined yet, I tried its use as a "value constraint" that is reusable. thoughts on this way of implementing? @satra @sanuann @akeshavan